tests/core/interceptor.c

```diff
@@ -56,7 +56,9 @@ TESTLIST_BEGIN (interceptor)
   TESTENTRY (already_replaced)
 # ifndef HAVE_ASAN
```

Comment:
```suggestion
#ifndef HAVE_ASAN
```

(Just spotted this existing style bug.)